user_docs/en/userGuide.t2t

```diff
@@ -39,9 +39,7 @@ Major highlights include:
 -
 
 ++ System Requirements ++[SystemRequirements]
-- Operating Systems: all 32-bit and 64-bit editions of Windows 7, Windows 8, Windows 8.1, Windows 10, Windows 11, and all Server Operating Systems starting from Windows Server 2008 R2.
-  - For Windows 7, NVDA requires Service Pack 1 or higher.
-  - For Windows Server 2008 R2, NVDA requires Service Pack 1 or higher.
+- Operating Systems: all 32-bit and 64-bit editions of Windows 8.1, Windows 10, Windows 11, and all Server Operating Systems starting from Windows Server 2012.

```

Comment:
`winVersion.isSupportedOS` should also be updated. Note that doing so might be slightly tricky, as we need to differentiate between client version of Windows (where Windows 8.1 is NT 6.3), and server (where Server 2012 is NT 6.2).

Reply:
Actually, isn't this a mistake, i.e. shouldn't the minimum server os be 2012R2?

```suggestion
- Operating Systems: all 32-bit and 64-bit editions of Windows 8.1, Windows 10, Windows 11, and all Server Operating Systems starting from Windows Server 2012 R2.
```

Reply:
That's correct, it should be R2 and Win 6.3.